API.md

```diff
@@ -3015,7 +3017,7 @@ Each request object includes the following properties:
   different response. Contains `null` when no response has been set (e.g. when a request terminates
   prematurely when the client disconnects).
 - `preResponses` - same as `pre` but represented as the response object created by the pre method.
-- `query` - an object containing the query parameters.
+- `query` - an object containing the parsed [query parameters](https://nodejs.org/docs/latest/api/url.html#url_urlobject_query).
```

Comment:
Not really, you can set the query object yourself (see https://github.com/hapijs/hapi/blob/master/API.md#requestseturlurl-striptrailingslash) which means it is not UrlObject.query per se. By default it will be though.